src/core.jl

```diff
@@ -109,6 +111,24 @@ struct Polygon <: Drawable
     vertices::Vector{Point}
 end
 
+"""
+    boundaryfill(x::Int, y::Int, fill_color::T, boundarycolor::T)
+
+
+"""
+struct boundaryfill{T<:Colorant} <: AbstractPolyFillAlgorithm
```

Comment:
struct and module should normally use the CapWords convention.

So it should be `BoundaryFill` here.